### ISSUE TITLE
NVDA not reading messages in Windows Live Messenger

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#251]
**Reported by jrutkowski on 11 Sep 2008 12:31 UTC**
When the windows live messenger has focus, NVDA fails to read both incoming and outgoing messages.

**Comment by nvdaTrac** [comment]
**Comment by mdcurran on 16 Feb 2009 04:49 UTC**
Is this true in latest snapshots? what language were you using? what is the name that NVDA reports of the history window where the messages are appearing?

**Comment by nvdaTrac** [comment]
**Comment by jteh on 1 May 2009 07:15 UTC**
This works for me. Please reopen if the problem persists and provide more information, including steps to reproduce and version information for all related software.
